src/map/handler/Map.PinchZoom.js

```diff
@@ -17,7 +16,7 @@ Map.mergeOptions({
 	// passed `'center'`, it will zoom to the center of the view regardless of
 	// where the touch events (fingers) were. Enabled for touch-capable web
 	// browsers.
```

Comment:
This comment is now out of date, as this feature is now enabled by default everywhere.